### DISCUSSION TITLE
Bootstrap 5 CheatSheet for classes, variables and mixins

### DISCUSSION THREAD

**Discussion by ajaypatelaj** [twbs/bootstrap#34167] [Show & Tell]
Hi hunters! 👋

We're really excited to finally release [Bootstrap 5 CheatSheet](https://bootstrap-cheatsheet.themeselection.com/) By [ThemeSelection](https://themeselection.com/)! 🎉

It is an interactive list of Bootstrap 5 classes, variables, and mixins.🎁 It helps you easily find the differences between Bootstrap 4 and Bootstrap 5🎊

The only Bootstrap 5 CheatSheet you will ever need whether you are upgrading your Bootstrap 4 project or starting a new Bootstrap 5 project.

### 🔗 Useful links:

Website: [https://bootstrap-cheatsheet.themeselection.com/](https://bootstrap-cheatsheet.themeselection.com/)
GitHub: [https://github.com/themeselection/bootstrap-cheatsheet](https://github.com/themeselection/bootstrap-cheatsheet)

### 💡  Features

👉🏻  Search for your class, variable, or mixin
👉🏻  Highlight Difference Between Bootstrap 4 & 5 Classes
👉🏻  Live preview of HTML & code snippet
👉🏻  Update code snippet on the go
👉🏻  Easily copy the code snippet and use it

Check our live tool [https://github.com/themeselection/bootstrap-cheatsheet](https://github.com/themeselection/bootstrap-cheatsheet) contribute and share your ideas, suggestions & feedback.

Stay Safe 🙏

**Comment by alokVishu** [comment]
Amazing Work...!!!!
